android/app/src/main/java/org/uwaterloo/subletr/navigation/NavigationFunctions.kt

```diff
@@ -5,5 +5,5 @@ import androidx.navigation.NavDestination.Companion.hierarchy
 
 fun NavDestination?.isTopLevelDestinationInHierarchy(destination: NavigationDestination) =
 	this?.hierarchy?.any {
-		it.route?.contains(destination.rootNavPath, true) ?: false
+		it.route?.equals(destination.rootNavPath, true) ?: false
```

Comment:
`"createaccount"` was being counted as `"account"` and the bottom bar was showing up in the create account screen

not sure if there are corner cases with this but I think `.equals` should be fine

Reply:
Yeah, should be fine.
I was looking to go back and fix that at some point anyways (when we implement chat)